vendors/circom/prover_main.cc

```diff
@@ -240,6 +243,15 @@ int RealMain(int argc, char** argv) {
     tachyon_cerr << "BLS12_381 curve is not supported on CUDA" << std::endl;
     return 1;
   }
+
+  if (!options.verify) {
+    std::cout
+        << "Icicle MSM/NTT may produce incorrect results if the polynomial "
```

Comment:
1. I'm not sure if we need to directly mention about Icicle since if with sufficient GPU memory, the results are correct.
2. "May produce incorrect results" seems to vague compared to what you said in the meeting. Let's express it more clearly so that users can understand more clearly that Icicle doesn't crash on OOM but instead returns incorrect results so verification might fail.